core/primitives/src/version.rs

```diff
@@ -238,6 +242,7 @@ impl ProtocolFeature {
             ProtocolFeature::Ed25519Verify
             | ProtocolFeature::ZeroBalanceAccount
             | ProtocolFeature::DelegateAction => 59,
+            ProtocolFeature::ComputeCosts => 61,
```

Comment:
When (or before) we can put stable protocol features to version 61, we have to bump `STABLE_PROTOCOL_VERSION` to 61. We usually do it in the same PR as the feature stabilization, so you can do it directly in this PR. That should also make stable tests easier.

Reply:
I've addressed this in https://github.com/near/nearcore/pull/8915